server/controllers/all.js

```diff
@@ -13,6 +15,9 @@ import { makeLogger, KError, asyncErr } from '../helpers';
 let log = makeLogger('controllers/all');
 
 const ERR_MSG_LOADING_ALL = 'Error when loading all Kresus data';
+const ENCRYPTION_ALGORITHM = 'aes-256-ctr';
+const PASSPHRASE_VALIDATION_REGEXP = /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{8,}$/;
```

Comment:
Maybe we could make alternatives such:
```
(complex short passphrase | simple long passphrase | medium intermediate passphrase)
```

_Disclaimer: Eyebleeds can occur after writing such things._